kubernetes/services/route_manager.yaml

```diff
@@ -24,6 +24,8 @@ data:
     microservices.port.service.url=http://portmanager-service.default.svc.cluster.local:9006
     protobuf.goal-state-message.version = 102
     ignite.thin.client.enable=true
+    ignite.kubeNamespace=ignite-alcor
+    ignite.kubeServiceName=ignite-alcor-service
```

Comment:
Do we need standalone DB for RM?

Reply:
We optimize RM code first? Since ignite DB deployed in Kubernetes can scale and it might be update to Hazelcast.

Reply:
Sounds good to me.